tools/filters/gff/gff_filter_by_feature_count.py

```diff
@@ -155,7 +155,7 @@ def __main__():
                 count += 1
         eval_text = '%s %s' % ( count, condition )
         if not check_expression(eval_text):
-            print("Invalid condition: %s, cannot filter." % condition, file=sys.stderr)
+            print "Invalid condition: %s, cannot filter." % condition, file=sys.stderr
```

Comment:
If you want Python2 syntax, then it should be:
```
print >> sys.stderr, "Invalid condition: %s, cannot filter." % condition
```
**But** it will create conflicts when forward merging. Better to backport the Py3 syntax with `import __future__` maybe.